db/migrate/20230705014229_create_goals.rb

```diff
@@ -0,0 +1,13 @@
+class CreateGoals < ActiveRecord::Migration[7.0]
+  def change
+    create_table :goals do |t|
+      t.references :user, null: false, foreign_key: true
+      t.integer :calorie_goal
+      t.integer :current_calorie
+      t.integer :nutrients
```

Comment:
im not sure what this should be, i think maybe a string??
since it would look like "protein : 20g ,calories: 200kcal..."

Reply:
we can add unit in json or front end. interger will help us do some calculation